src/sentry/db/postgres/__init__.py

```diff
@@ -0,0 +1 @@
+from .lookups import *  # NOQA
```

Comment:
@asottile-sentry are we moving away from re-exports like this?

Reply:
I had no better idea where to put this so that is picked up during start up, and that it gets registered on an `ArrayField`

Reply:
You could put this lookup with our ArrayField in `sentry.db.models.fields.array` :shrug:

Reply:
@markstory maybe that would also make sense, an to register it also to our ArrayField, but I need to use it on Django's `ArrayField` for Postgres. More specifically on `DashboardWidgetQuery` model (`aggregates` field) from `src/sentry/models/dashboard_widget.py`.

Would it be weird to register it also to Django's `ArrayField` in the same file/place where we define our own `ArrayField`?

I have no strong opinions here, and I am open to any proposal how to make it better and more in line with our codebase

Reply:
yeah I just killed the star import here, would be a shame to bring it back.  I think there are better places for database initialization throughout `sentry.db` as mark points out

Reply:
I have moved it to `src/sentry/db/models/fields/array.py` as Mark suggested and registered this custom lookup on both our custom `ArrayField` and Django's `ArrayField`